tests/AbstractTest.php

```diff
@@ -5,33 +5,39 @@
 abstract class AbstractTest extends \PHPUnit_Framework_TestCase
 {
     /**
+     * Twig environmnt object.
+     *
+     * @var \Twig_Environment
+     */
+    protected $twig;
+
+    /**
      * Provides a mocked Configuration object.
      */
     public function getConfig()
     {
-        $logger = $this->getMockBuilder('Psr\Log\LoggerInterface')
-            ->getMock();
+        $twig = $this->getTwig();
+        $retriever = new \JsonSchema\Uri\UriRetriever();
+        $logger = new \Psr\Log\NullLogger();
+        $retriever = new \JsonSchema\Uri\UriRetriever();
```

Comment:
It gets passed into the resolver on the next line.

`$resolver = new \JsonSchema\RefResolver($retriever);`